specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/examples/DatabaseExtendedAuditingSettingsList.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "blobauditingtest-6852",
+    "serverName": "blobauditingtest-2080",
+    "databaseName": "testdb",
+    "api-version": "2017-03-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/blobauditingtest-6852/providers/Microsoft.Sql/servers/blobauditingtest-2080/databases/testdb",
+            "name": "default",
+            "type": "Microsoft.Sql/servers/databases/extendedAuditingSettings",
```

Comment:
>Microsoft.Sql/servers/databases/extendedAuditingSettings [](start = 21, length = 56)

The resource type does not match the ID.

Reply:
fixed

Reply:
The reosurce type is `Microsoft.Sql/servers/databases/extendedAuditingSettings`, which implies a resource id that looks something like this:
`/subscriptions/{id}/resourceGroups/{rg}/providers/Microsoft.Sql/servers/{serverName}/databases/{db}/extendedAuditingSettings/default`

---
In reply to: [402364988](https://github.com/Azure/azure-rest-api-specs/pull/8897#discussion_r402364988) [](ancestors = 402364988)

Reply:
please see commit https://github.com/Azure/azure-rest-api-specs/pull/8897/commits/569fa9e5ee245e8264cca11bd698e933c97a7b1a

Reply:
Yes, that is the one I'm looking at. Your `id` has `.../databases/testdb/default`. It should be `.../databases/testdb/extendedAuditingSettings/default`

---
In reply to: [402933615](https://github.com/Azure/azure-rest-api-specs/pull/8897#discussion_r402933615) [](ancestors = 402933615)